### ISSUE TITLE
sorting the marketplace courses

### ISSUE THREAD

**Issue by kirkoplerno** [webhat/oplerno#152]
just like on a retail website, you can sort by price (low to high, high to low) type (shirt, pants, etc) color etc.

can we have a few links at the top (or added to the browse menu) that allow users to sort by start time?  any other ideas?

**Comment by sushimustwrite** [comment]
Yes yes yes. This needs to happen. Sorting by price and subject would also
be good to see.

**Comment by kirkoplerno** [comment]
@sushimustwrite we've definitely chatted about the subject ability.  it's going to be nuanced since we dont have 'departments' but perhaps we could allow faculty to choose up to three 'tags' that would allow their course to come up as a search result within a subject
